common/src/main/java/com/wynntils/screens/settings/elements/BooleanConfigOptionElement.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.settings.elements;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.config.ConfigHolder;
+import com.wynntils.mc.render.RenderUtils;
+import com.wynntils.mc.render.Texture;
+import com.wynntils.screens.settings.widgets.FeatureSettingWidget;
+
+public class BooleanConfigOptionElement extends ConfigOptionElement {
+    public BooleanConfigOptionElement(ConfigHolder configHolder, FeatureSettingWidget featureSettingWidget) {
+        super(configHolder, featureSettingWidget);
+    }
+
+    @Override
+    protected void renderConfigAppropriateButton(PoseStack poseStack, float width, float height) {
+        if (configHolder.getType().equals(Boolean.class)) {
```

Comment:
Isn't this unnecessry?

Reply:
It is now. There was a need in another iterations.